collection_test.go

```diff
@@ -69,4 +70,87 @@ func TestCollectionScripts(t *testing.T) {
 		assert.JSONEq(otu.T, expected, result)
 
 	})
+
+	t.Run("Should be able to mint Dandy and then get it by script Index", func(t *testing.T) {
+
+		expected := `
+		[
+			"findDandy/81"
+		,
+			"findDandy/82"
+		,
+			"findDandy/80"
+		]
+		`
+
+		otu := NewOverflowTest(t)
+		otu.setupFIND().
+			setupDandy("user1").
+			registerDandyInNFTRegistry().
+			mintThreeExampleDandies()
+
+		result := otu.O.ScriptFromFile("collectionIndex").
+			Args(otu.O.Arguments().Address("user1")).
+			RunReturnsJsonString()
+		assert.JSONEq(otu.T, expected, result)
+
+	})
+
+	t.Run("Should be able to query the Dandies by resolve partial collection scripts", func(t *testing.T) {
+
+		expected := `
+		[{
+			"collection": "Dandy",
+			"contentType": "image",
+			"id": "80",
+			"image": "https://neomotorcycles.co.uk/assets/img/neo_motorcycle_side.webp",
+			"metadata": {},
+			"name": "Neo Motorcycle 1 of 3",
+			"rarity": "",
+			"typeIdentifier": "A.f8d6e0586b0a20c7.Dandy.NFT",
+			"url": "find.xyz",
+			"uuid": "80"
+		},
+		{
+			"collection": "Dandy",
+			"contentType": "image",
+			"id": "81",
+			"image": "https://neomotorcycles.co.uk/assets/img/neo_motorcycle_side.webp",
+			"metadata": {},
+			"name": "Neo Motorcycle 2 of 3",
+			"rarity": "",
+			"typeIdentifier": "A.f8d6e0586b0a20c7.Dandy.NFT",
+			"url": "find.xyz",
+			"uuid": "81"
+		},
+		{
+			"collection": "Dandy",
+			"contentType": "image",
+			"id": "82",
+			"image": "https://neomotorcycles.co.uk/assets/img/neo_motorcycle_side.webp",
+			"metadata": {},
+			"name": "Neo Motorcycle 3 of 3",
+			"rarity": "",
+			"typeIdentifier": "A.f8d6e0586b0a20c7.Dandy.NFT",
+			"url": "find.xyz",
+			"uuid": "82"
+		}]
+		`
+
+		otu := NewOverflowTest(t)
+		otu.setupFIND().
+			setupDandy("user1").
+			registerDandyInNFTRegistry().
+			mintThreeExampleDandies()
+
+		result := otu.O.ScriptFromFile("resolvePartialCollection").
```

Comment:
The input to this must be collection names not paths, we can look up the path in the NFTRegistry.